src/deploy/index.ts

```diff
@@ -133,4 +134,4 @@ export async function proposal(actions: ProposalAction[], description: string):
     }
   }
   return [targets, values, signatures, calldatas, description];
-}
\ No newline at end of file
+}
```

Comment:
Ah yeah, this was brought up from  /c3-api indentation setting.  😅 
🧇 brought up adding new line at the end of the files are the correct style via: https://thoughtbot.com/blog/no-newline-at-end-of-file

Too easy to mixed up the styles for VSC lol, since both are typescript, which format setting will always apply one setting that's active on typescript.

Let me change the setting back to default (no new line at the end), and re-push the files to remove those new line.